server/api/topics.js

```diff
@@ -58,4 +59,18 @@ router.get("/:topic_id/questions", (req, res) => {
 			res.json(500);
 		});
 });
+router.post("/:topic_id/createQuestion", (req, res) => {
```

Comment:
Can you change that to `/:topid_id/questions`? Better make the change in here ;)

Reply:
Done 
change the name to /:topid_id/question